frontend/packages/dev-console/src/components/topology/TopologySideBar.tsx

```diff
@@ -27,12 +34,12 @@ function metadataUIDCheck(items: any): ResourceProps[] {
 const TopologySideBar: React.FC<TopologySideBarProps> = ({ item, show, onClose }) => {
   let itemtoShowOnSideBar;
   if (item) {
-    const dc = item.resources.filter(
-      (o) => o.kind === 'DeploymentConfig' || o.kind === 'Deployment' || o.kind === 'DaemonSet',
+    const dc = item.resources.filter(({ kind }) => possibleKinds.includes(kind as KINDS));
```

Comment:
you do not necessarily need another variable to hold the values`possibleKinds`, you could use 
```suggestion
    const dc = item.resources.filter(({ kind }) => Object.values(KINDS).includes(kind as KINDS));
```

Reply:
Actually, Here I do need that If you see enum KINDS I added Route, Services, etc to it.